### ISSUE TITLE
Interstitials work fine, banners not showing

### ISSUE THREAD

**Issue by levelup-fx** [floatinghotpot/cordova-admob-pro#193]
Hello,

I am creating a simple test app in Ionic / Cordova and I am using your Admob Pro plugin to display ads.

Interstitial ads show up as intended. However banners don't show up no matter what changes I make to the AdMob.createBanner() method.

I am using Genymotion to test - Could that be the problem?

Here is my code below:

<pre>
$scope.showBanner = function()
  {
    var admobid = {};

    if(/(android)/i.test(navigator.userAgent) ) 
    {
        admobid = 
        {
            banner: 'ca-app-pub-XXXXXXXXXXX/YYYYYYYYYY',
            interstitial: 'ca-app-pub-XXXXXXXXXXX/AAAAAAAAAAAAAA'
        };
    } else if(/(ipod|iphone|ipad)/i.test(navigator.userAgent)) 
    {
        admobid = 
        {
            banner: 'ca-app-pub-XXXXXXXXXXX/CCCCCCCCCCCC',
            interstitial: 'ca-app-pub-XXXXXXXXXXX/ZZZZZZZZZZZZZZ'
        };
    } 

    if(window.AdMob)
    {
        window.AdMob.createBanner({
        adId: admobid.banner,
        position: window.AdMob.AD_POSITION.BOTTOM_CENTER,
        autoShow: true});
    }
    else
    {
        //alert("Admob plugin not present");
        console.log("Admob plugin not present");
    }
 }
</pre>

**Comment by floatinghotpot** [comment]
I did not try Genymotion, not sure what it did. Some tools may change the cordova view hierarchy and cause problem.

Maybe you can try adding one options to createBanner(), overlap:true

**Comment by levelup-fx** [comment]
Hello Raymond,

thanks for the response. 

I found out what the problem was. Well, sort of.

I'm writing the code in AngularJS and I was calling createBanner() on the main App Controller.

For some reason banners don't show up when I call the function from there there.

I called createBanner on a second, random controller and it works fine... Now I have to figure out it doesn't work with the main controller.
